instrumentation/otel-webserver-module/src/apache/ApacheHooks.cpp

```diff
@@ -516,6 +520,52 @@ void fillRequestPayload(request_rec* request, appd::core::RequestPayload* payloa
     val = request->method ? request->method: " ";
     payload->set_http_request_method(val);
 
+    // websrv-698 Setting server span attributes
+    payload->set_status_code(request->status);
+
+    if (request->server)
+    {
+        payload->set_server_name(request->server->server_hostname);
+    }
+    payload->set_scheme(ap_run_http_scheme(request));
+
+    if (request->hostname)
+    {
+        payload->set_host(request->hostname);
+    }
+
+    if (request->unparsed_uri)
+    {
+        payload->set_target(request->unparsed_uri);
+    }
+
+    switch (request->proto_num)
+    {  // TODO: consider using ap_get_protocol for other flavors
+      case HTTP_PROTO_1000:
+        payload->set_flavor(kHTTPFlavor1_0.c_str());
+        break;
+      case HTTP_PROTO_1001:
+        payload->set_flavor(kHTTPFlavor1_1.c_str());
+        break;
+    }
+
+#ifdef APLOG_USE_MODULE
+
+    payload->set_client_ip(request->useragent_ip);
+    if (request->connection)
+    {
+        payload->set_net_ip(request->connection->client_ip);
+    }
+
+#else
+    if (request->connection)
+    {
+        payload->set_net_ip(request->connection->remote_ip);
+    }
+
+#endif
+
+    payload->set_port(ap_default_port(request));
```

Comment:
Same as above

Reply:
/**
 * Get the default port for a request (which depends on the scheme).
 * @param r The request
 */
#define ap_default_port(r)      ap_run_default_port(r)